From 7102fb03239b8769f3436495a9488210fe79ea6f Mon Sep 17 00:00:00 2001
From: Gwenael Treuveur <gwenael.treuveur@foss.st.com>
Date: Thu, 12 Sep 2024 17:36:52 +0200
Subject: [PATCH] arm64: dts: st: update rproc for Cortex-M0 in stm32mp257f-ev1

Update peripherals clocks used by Cortex-M0.

Change-Id: I9eb695d8b7bf7715232c0a214ce654caf6d4cac1
Signed-off-by: Gwenael Treuveur <gwenael.treuveur@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/404453
Domain-Review: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Gwenael TREUVEUR <gwenael.treuveur@st.com>
Reviewed-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Gwenael TREUVEUR <gwenael.treuveur@st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -614,8 +614,13 @@
 	mboxes = <&ipcc2 0>, <&ipcc2 1>, <&ipcc2 2>;
 	mbox-names = "rx", "tx", "shutdown";
 	memory-region = <&cm0_cube_fw>, <&cm0_cube_data>;
-	clocks = <&rcc CK_CPU3>, <&rcc CK_KER_LPUART1>;
-	clock-names = "c3_clk", "lpuart1_clk";
+	clocks = <&rcc CK_CPU3>,
+		 <&rcc CK_CPU3_AM>,
+		 <&rcc CK_LPUART1_C3>,
+		 <&rcc CK_KER_LPUART1>,
+		 <&rcc CK_LPUART1_AM>,
+		 <&scmi_clk CK_SCMI_IPCC2>,
+		 <&scmi_clk CK_SCMI_IPCC2_AM>;
 	status = "okay";
 };
 
